Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.cpp

```diff
@@ -1007,10 +1007,11 @@ IGNORE_WARNINGS_END
     }), role.isolatedCopy().releaseImpl().leakRef(), static_cast<GClosureNotify>([](gpointer userData, GClosure*) {
         reinterpret_cast<StringImpl*>(userData)->deref();
     }), static_cast<GConnectFlags>(0));
+    ASSERT(g_object_is_floating(audioSink));
     return audioSink;
 }
 
-GstElement* createPlatformAudioSink(const String& role)
+GstElement* /* (transfer floating) */ createPlatformAudioSink(const String& role)
```

Comment:
No `ASSERT(g_object_is_floating(audioSink));` here?

Reply:
It is already in the two constructors used inside. If you think I should add it here anyway, I can do that too.

Reply:
Ah OK, I think it's not needed then :) r+